ONSdigital/ras-rm-sample-file-uploader
# fix duplicate sample unit ID bug

# What and why?
This PR provides an efficient fix for the sample unit duplicate bug. Now, files with a duplicate sample unit will be rejected with an error redirect if the user tries to enter them. A unit test has been created that hits this error, and the test sample unit files have been modified so that the other tests are unaffected.

# How to test?
On R-ops, try to upload a sample file that has a duplicate sample unit ID in it. It should fail almost instantly and redirect the user back to the previous page with a little red box that tells the user there's a duplicate sample in the file.
Also, check that valid sample files (ones that don't have duplicate sample unit IDs) can still be uploaded, even if the sample unit IDs are not in numerical order in the file.

# Trello
[Card](https://trello.com/c/upvtJmqp)
